<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabsubtitle</class>
 <widget class="QWidget" name="tabsubtitle">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_general_subtitles">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>220</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtitle settings:</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_subtitle_look">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>220</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Subtitle look:</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>240</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QGraphicsView" name="subtitle_test_graphics_view">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>740</width>
     <height>150</height>
    </rect>
   </property>
   <property name="transformationAnchor">
    <enum>QGraphicsView::NoAnchor</enum>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>411</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <property name="labelAlignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <widget class="QCheckBox" name="box_no_sub">
      <property name="text">
       <string>no-sub</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="box_sub_delay">
      <property name="text">
       <string>sub-delay</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="value_sub_delay">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="box_sub_scale_by_window">
      <property name="accessibleName">
       <string>sub-scale-by-window</string>
      </property>
      <property name="text">
       <string>sub-scale-by-window</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="box_ass_scale_with_window">
      <property name="accessibleName">
       <string>ass-scale-with-window</string>
      </property>
      <property name="text">
       <string>ass-scale-with-window</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="box_no_embedded_fonts">
      <property name="text">
       <string>no-embeddedfonts</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="box_ass_hinting">
      <property name="text">
       <string>ass-hinting</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QComboBox" name="comboBox_ass_hinting">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>none</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>light</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>native</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="box_ass_sharper">
      <property name="text">
       <string>ass-shaper</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="comboBox_ass_sharper">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>simple</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>complex</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="box_ass_force_style">
      <property name="text">
       <string>ass-force-style</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="value_ass_force_style">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="box_ass_style_override">
      <property name="text">
       <string>ass-style-override</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QComboBox" name="comboBox_ass_override">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>force</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="box_sub_auto">
      <property name="text">
       <string>sub-auto</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QComboBox" name="comboBox_sub_auto">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>exact</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>fuzzy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>all</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QCheckBox" name="box_stretch_dvd_subs">
      <property name="accessibleName">
       <string>stretch-dvd-subs</string>
      </property>
      <property name="text">
       <string>stretch-dvd-subs</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QCheckBox" name="box_no_sub_fix_timing">
      <property name="text">
       <string>no-sub-fix-timing</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QCheckBox" name="box_sub_speed">
      <property name="text">
       <string>sub-speed</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLineEdit" name="value_sub_speed">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QCheckBox" name="box_demuxer_mkv_subtitle_preroll">
      <property name="text">
       <string>demuxer-mkv-subtitle-preroll</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="box_sub_scale_with_window">
      <property name="accessibleName">
       <string/>
      </property>
      <property name="text">
       <string>sub-scale-with-window</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QCheckBox" name="box_slang">
      <property name="text">
       <string>slang</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLineEdit" name="valuebox_slang">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="comboBox_sub_scale_by_window">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="comboBox_sub_scale_with_window">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBox_ass_scale_with_window">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QComboBox" name="comboBox_stretch_dvd_subs">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <item>
       <property name="text">
        <string>yes</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>no</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>493</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="horizontalSpacing">
     <number>30</number>
    </property>
    <property name="verticalSpacing">
     <number>15</number>
    </property>
    <item row="7" column="0">
     <widget class="QCheckBox" name="box_sub_text_font_color">
      <property name="text">
       <string>sub-text-color</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label_sub_text_font_size_counter">
      <property name="text">
       <string>55</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="box_sub_text_border_color">
      <property name="text">
       <string>sub-text-border-color</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QPushButton" name="button_pick_color_font_color">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>://images/colorpicker.png</normaloff>://images/colorpicker.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="box_sub_text_font">
      <property name="text">
       <string>sub-text-font</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="box_sub_text_font_size">
      <property name="text">
       <string>sub-text-font-size</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_sub_text_border_size_counter">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QSlider" name="horizontalSlider_sub_text_font_shadow_offset">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QSlider" name="horizontalSlider_sub_text_font_border_size">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>25</number>
      </property>
      <property name="sliderPosition">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="value_sub_text_font_color">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>#000000</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_sub_text_shadow_offset_counter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QCheckBox" name="box_sub_text_font_border_size">
      <property name="text">
       <string>sub-text-border-size</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QSlider" name="horizontalSlider_sub_text_font_size">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>150</number>
      </property>
      <property name="sliderPosition">
       <number>55</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="box_sub_text_font_shadow_offset">
      <property name="text">
       <string>sub-text-shadow-offset</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QCheckBox" name="box_sub_text_shadow_color">
      <property name="text">
       <string>sub-text-shadow-color</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="value_sub_text_border_color">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>#000000</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="value_sub_text_shadow_color">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>#000000</string>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="button_pick_color_border_color">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>://images/colorpicker.png</normaloff>://images/colorpicker.png</iconset>
      </property>
     </widget>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="button_pick_color_shadow_color">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>://images/colorpicker.png</normaloff>://images/colorpicker.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFontComboBox" name="fontComboBox_sub_text_font">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="valuebox_sub_text_font">
   <property name="geometry">
    <rect>
     <x>1500</x>
     <y>10</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>box_sub_text_font_color</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_pick_color_font_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>569</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>570</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_sub_text_border_color</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_pick_color_border_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>599</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>600</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_sub_text_shadow_color</sender>
   <signal>toggled(bool)</signal>
   <receiver>button_pick_color_shadow_color</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>629</y>
    </hint>
    <hint type="destinationlabel">
     <x>760</x>
     <y>630</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
